Sources/CodegenCLI/Commands/Initialize.swift

```diff
@@ -11,11 +11,21 @@ public struct Initialize: ParsableCommand {
     abstract: "Initialize a new configuration with defaults."
   )
 
+  @Option(
+    name: [.long],
+    help: "DEPRECATED - Use --schema-namespace instead."
+  )
+  // When removing this property also do:
```

Comment:
I'm not a huge fan of needing to have this here but some of the changes for the CLI option deprecation warning feel a bit hacky and I want to make sure we get rid of them when we finally remove the deprecated property.